bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_proc_card.dat

```diff
@@ -0,0 +1,11 @@
+define p = g u c d s u~ c~ d~ s~ b b~ 
+define j = g u c d s u~ c~ d~ s~ b b~ 
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+
+import model sm
+import model ALP
+
+generate p p > h > Z alp , Z > l+ l- , alp > a a
+
+output ALP -nojpeg 
```

Comment:
Are you considering only m(alp)=20 scenario?

Reply:
We are considering ALP mass range [2GeV, 35GeV] and take 1GeV as the step.

Reply:
Can you make add the script which generates the cards for other mass settings?

Reply:
@sihyunjeon  Add 'copy.sh' for all mass settings. Please take a look at it. Thank you.